scripts/build_wasm.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+
+if [[ -z "$1" ]]; then
+  echo "Need to pass a parachain name as argument"
+  exit 1
+fi
+
+parachain_name=$1
+
+echo "🏭 building runtime wasm..."
+
+cargo build \
+  --locked \
+  --profile release \
+  --package ${parachain_name}-parachain-runtime \
+  --target-dir target/ \
+  -Z unstable-options
+
+used_wasm=target/release/wbuild/${parachain_name}-parachain-runtime/${parachain_name}_parachain_runtime.compact.compressed.wasm
+
+echo "🔢 hashing ${parachain_name}_parachain_runtime.compact.compressed.wasm..."
+wasm_info=$(subwasm info --json $used_wasm)
```

Comment:
Given that all these Substrate tools are pretty immature let's include downloading a versioned `subwasm` in this script.

Reply:
@AwesomeIbex asked me to remove it and install it on github runners.
For me both ways are fine.
Thoughts?

Reply:
I have brought that back because epyc fails on this step each time saying `subwasm` is missing even tho its in .cargo, /usr/local/bin and /usr/bin.
Let's remove it after we will have set of unified github runners.